weak_libjack.c

```diff
@@ -53,7 +53,7 @@ static void* lib_symbol(void* const lib, const char* const sym) {
 #endif
 }
 
-#if _MSC_VER && !__INTEL_COMPILER
+#if defined(_MSC_VER) && _MSC_VER && !__INTEL_COMPILER
```

Comment:
This doesn't look right  `__INTEL_COMPILER` is now only checked if `_MSC_VER` is defined.